rust/crd/src/lib.rs

```diff
@@ -261,6 +264,13 @@ pub struct S3Spec {
     pub endpoint: Option<String>,
 }
 
+#[derive(Clone, Debug, Default, Deserialize, JsonSchema, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct OpaSpec {
+    pub opa_config_map_name: String,
+    pub opa_druid_api_path: String,
+}
```

Comment:
We just removed this in Kafka. I would only go for the config map name. The api path will be fixed in code or may be overriden via overrides if neccessary (but we can just document whats the default path should be)